benchmarks/base.py

```diff
@@ -54,9 +67,7 @@ def random_data_dir(num_files, file_size):
 
     if not os.path.exists(dir_path):
         os.makedirs(dir_path, exist_ok=True)
-        filenames = [
-            os.path.join(dir_path, "file_{}".format(str(i))) for i in range(num_files)
-        ]
-        with ThreadPoolExecutor(max_workers=max(cpu_count() / 2, 2)) as executor:
-            executor.map(random_file, filenames, len(filenames) * [file_size])
+        fnames = [f"{dir_path}{os.sep}file_{str(i)}" for i in range(num_files)]
```

Comment:
Please use `join`, there is no need to optimize benchmarks that much :slightly_smiling_face: